src/SplitPane.vue

```diff
@@ -38,12 +41,20 @@ function dragMove(e: MouseEvent) {
       : container.value.offsetWidth
     const dp = position - startPosition
     state.split = startSplit + +((dp / totalSize) * 100).toFixed(2)
+
+    state.viewHeight = rightView.value.offsetHeight - 38
```

Comment:
Hmm, I don't think this magic constant is good enough. It's better to get the iframe element directly.

Reply:
Done.